src/network/tarcap/packets_handlers/vote_packets_handler.cpp

```diff
@@ -182,7 +182,7 @@ inline void VotePacketsHandler::processPbftNextVotesPacket(const dev::RLP &packe
       std::vector<Vote> send_next_votes_bundle;
       for (auto const &v : next_votes) {
         if (!peer_to_share_to.second->isVoteKnown(v.getHash())) {
-          send_next_votes_bundle.emplace_back(v);
+          send_next_votes_bundle.push_back(std::move(v));
```

Comment:
btw Vote class does not support move semantics. This is true for almost all(if not literally all) of our basic classes like Transaction, DagBlock, etc... So we are in fact doing copies almost everywhere. We should refactor those classes to actually gain something by using std::move